Amir's SleepBuddy project.md

```diff
@@ -0,0 +1,54 @@
+
+The SleepBuddy can also :
+
+- Give the room temperature 🌡️
+- Turn on light (night light) when it detects something moving 🐕🐈🚶
+- Display 👀 the music you are listening to and the time thanks to its screens.
+- Share information about your sleep with your friend's SleepBuddy (cross project with @CodingForAddicts) thanks to its wifi 🛜 antenna.
+
+## ✅ Conditions to submit proposal:
+
+- [x] Be a current high school student 🧑‍🎓 (or younger)
+- [x] Submit an RSVP at https://hackclub.com/winter
+- [x] Be a member of the Hack Club Slack (https://hackclub.com/slack)
+- [x] Review my project proposal
+
+## 📃 Plan:
+1. Buy the hardware 💰
+2. Print the prototype of the box in a local FabLab
+3. Create the final box in polypropylene and wood
+4. Program 💻 my arduino with all the components (uses in the summary)
+5. Put all the components together in the box
+
+## 💸 Budget:
+
+| Product   | Supplier/Link  | Cost (including some taxes) |
+|:--|:--|:--|
+| Arduino UNO REV 2 WIFI/Bluetooth | [arduino](https://store.arduino.cc/products/arduino-uno-wifi-rev2?variant=35570905743511) | 55,90 $ (there is a promo code so I don't know the exact price) |
+| Stereo Enclosed Speaker - 3W 8Ω (x2) | [arduino](https://store.arduino.cc/products/stereo-enclosed-speaker-3w-8ω?variant=40119999103127) | 9,60 $ (there is a promo code so I don't know the exact price) |
+| NeoPixel LED strip - 30 RGB LEDs, 0,5m | [semageek](https://boutique.semageek.com/fr/1400-bande-de-led-neopixel-30-led-rgb-avec-connecteur-jst-05m-3007283931097.html?search_query=Bande+de+LED+NeoPixel+30+LED+RGB+avec+connecteur+JST+-+0%2C5m&results=355) | 15,93 $ |
+| Humidity and temperature sensor | [semageek](https://boutique.semageek.com/fr/138-dht11-capteur-de-temperature-et-humidite-digital-3004931765998.html?search_query=temperature&results=65) | 7,00 $ |
+| Grove - Touch Sensor (button on the top to stop the alarm) | [arduino](https://store.arduino.cc/products/grove-touch-sensor?variant=35573916991639) | 8,07 $ |
+| SUBTOTAL (arduino and semageek products + shipping cost)  |  | 107,64 $ |
+| Switch on-off (turn on and off the SleepBuddy) | [amazon](https://www.amazon.fr/ARTGEAR-Interrupteur-Bascule-Positions-Broches/dp/B07GDCNXKP/ref=sr_1_21?crid=D4O86S68WC7U&keywords=switch%2Barduino&qid=1671830001&sprefix=switch%2Barduin%2Caps%2C180&sr=8-21&th=1) | 10,61 $ |
+| Movement sensor | [amazon](https://www.amazon.fr/AZDelivery-HC-SR501-ultrasons-T%C3%A9l%C3%A9m%C3%A8tre-Raspberry/dp/B07CNBYRQ7/ref=sr_1_2?__mk_fr_FR=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=23F3OXHPOD84&keywords=movement%2Bsensor%2Barduino&qid=1671830580&sprefix=movement%2Bsensor%2Barduino%2Caps%2C87&sr=8-2&th=1) | 14,32 $ |
+| Jumper Wires | [amazon](https://www.amazon.fr/YXPCARS-câbles-jumper-femelle-Arduino/dp/B08HQ7K6M7/ref=sr_1_5?crid=TV7B8USTG35U&keywords=jumper+cable&qid=1671636526&sprefix=jumper+ca%2Caps%2C108&sr=8-5) | 11,66 $ |
+| wood and Polypropylene parts | Local FabLab (idk which one exactly) | max 50 $ |
```

Comment:
> (idk which one exactly)

Before we approve your project, could you decide which local FabLab you might use?